pkg/kudoctl/kudoinit/migration/migration.go

```diff
@@ -1,8 +1,13 @@
 package migration
 
-import "github.com/kudobuilder/kudo/pkg/kudoctl/kube"
+// Migrater is the base interface for migrations.
 
```

Comment:
Doesn't this line have to be removed for the comment above to document `Migrater`?

Reply:
Hmmm, good question. I've removed it, you're probably right.